### PR TITLE
Fix various docs typos

### PR DESCRIPTION
**Pull request by phil-davis**
I noticed a few typos, so I set my IDE to work finding more.

### CODE REVIEW

File: docs/release-notes/10-4-0/README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ In OpenProject you can now manage multiple building models in IFC format in para
 
 ![OpenProject-BIM-manage-IFC-models](OpenProject-BIM-manage-IFC-models.png)
 
-![OpenPoject-BIM_structural](OpenPoject-BIM_structural.png)
+![OpenProject-BIM_structural](OpenPoject-BIM_structural.png)
```
Comment (phil-davis) [inline docs/release-notes/10-4-0/README.md]: Note: I did not fix the png file name - that is a reference to an actual file, which has a typo in the name.

File: docs/release-notes/10-4-0/README.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ View full 3D objects in OpenProject in IFC format. Select and display the model
 
 You can change between a 3D view or 2D view of the building model in OpenProject.
 
-![OpenPoject-BIM_2D](OpenPoject-BIM_2D.png)
+![OpenProject-BIM_2D](OpenPoject-BIM_2D.png)
```
Comment (phil-davis) [inline docs/release-notes/10-4-0/README.md]: Note: I did not fix the png file name - that is a reference to an actual file, which has a typo in the name.

Let me know if you want the files renamed...

File: docs/user-guide/integrations/Nextcloud/README.md [hunk 3]
```diff
@@ -41,13 +41,13 @@ Enter the URL of your OpenProject instance and your access token which you can f
 
 **Display of OpenProject in the Nextcloud dashboard**
 
-On the Nextcloud dashboard you can add an OpenProject widget. Display the latest changes to your project's work packages to keep an eye on your ongoing project activites directly from your Nextcloud instance.
+On the Nextcloud dashboard you can add an OpenProject widget. Display the latest changes to your project's work packages to keep an eye on your ongoing project activities directly from your Nextcloud instance.
 
-![Add_OpenPorject_widget](../faq/Add_OpenPorject_widget.png)
+![Add_OpenProject_widget](../faq/Add_OpenPorject_widget.png)
```
Comment (phil-davis) [inline docs/user-guide/integrations/Nextcloud/README.md:46]: I did not change the file name. There is https://github.com/opf/openproject/blob/dev/docs/user-guide/integrations/Integrations-faq/Add_OpenPorject_widget.png - maybe that is where this file lives now?
There is no `faq` directory any more.

Maybe I can do a separate PR  to fixup links that might have got broken by https://github.com/opf/openproject/commit/38446ac060c4ad0e3139d2d8c5f0f1aba07d79b5

File: docs/user-guide/projects/project-settings/versions/README.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Also, version will  be used to enable the Backlogs module, i.e. to create a prod
 
 Press the green **+ Version** button to create a new version for a project.
 
-![User-guide-project-settins-versions](User-guide-project-settins-versions.png)
+![User-guide-project-settings-versions](User-guide-project-settins-versions.png)
```
Comment (phil-davis) [inline docs/user-guide/projects/project-settings/versions/README.md]: I did not change the file name. There seem to be 2 files:
```
User-guide-project-settings-versions.png
User-guide-project-settins-versions.png
```

The one with the typo seems to be later. It has Version 1.2 in the example.

I can make a separate PR to sort that out.

**Comment by phil-davis** [comment]
Sadly, CI failed. I don't think that I broke anything with these changes? Maybe intermittent tests?

**Comment by phil-davis** [comment]
> Sadly, CI failed. I don't think that I broke anything with these changes? Maybe intermittent tests?

PR #9359 has the same fails. So it is not related to the actual changes in this PR.

**Comment by oliverguenther** [comment]
@phil-davis Indeed this spec is flickering. I'm looking into fixing it at the moment.

File: docs/release-notes/10-4-0/README.md (hunk 1 above)
Comment (phil-davis) [reply docs/release-notes/10-4-0/README.md]: Done

File: docs/release-notes/10-4-0/README.md (hunk 2 above)
Comment (phil-davis) [reply docs/release-notes/10-4-0/README.md]: Done

File: docs/user-guide/projects/project-settings/versions/README.md (hunk 4 above)
Comment (phil-davis) [reply docs/user-guide/projects/project-settings/versions/README.md]: Done

**Comment by ulferts** [comment]
@phil-davis we worked really hard over the years to include that many typos. I feel a little embarrassed by the sheer number of them so thanks for fixing.

**Comment by phil-davis** [comment]
> @phil-davis we worked really hard over the years to include that many typos. I feel a little embarrassed by the sheer number of them so thanks for fixing.

No problem - lots of them are in old release notes :) - my IDE did most of the hard work.

**Comment by oliverguenther** [comment]
Thanks a lot again @phil-davis :tada:

File: docs/user-guide/integrations/Nextcloud/README.md (hunk 3 above)
Comment (phil-davis) [reply docs/user-guide/integrations/Nextcloud/README.md:46]: See #9370

**Pull request merged**